README.md

```diff
@@ -91,10 +91,10 @@ A simple wrapper for the Jaeger exporter of opentelemetry-java. gRPC is currentl
 
 | System property                   | Environment variable              | Description                                                                                                         |
 |-----------------------------------|-----------------------------------|---------------------------------------------------------------------------------------------------------------------|
-| otel.exporter=jaeger              | OTEL_EXPORTER=jaeger              | Select the Jaeger exporter                                                                                          |
+| otel.exporter=jaeger              | OTEL_EXPORTER=jaeger              | Optional: Select the Jaeger exporter for gRPC capable environments. This should not be set in a default Kubernetes setuip.                                                                                          |
```

Comment:
I wouldn't say "default" here. That makes a lot of assumptions about the world. I'd just mention that it's not compatible with the smart agent.

Reply:
It is compatible with SmartAgent on a host... just not in K8S.